dvc/repo/get.py

```diff
@@ -20,14 +20,14 @@ def __init__(self):
         )
 
 
-def get(url, path, out=None, rev=None, jobs=None):
+def get(url, path, out=None, rev=None, jobs=None, force=False):
     import shortuuid
 
     from dvc.dvcfile import is_valid_filename
     from dvc.external_repo import external_repo
     from dvc.fs.callbacks import Callback
 
-    out = resolve_output(path, out)
+    out = resolve_output(path, out, override=force, override_hint=True)
```

Comment:
override vs force

Would be more natural for me to add an `--override` flag, but I went with force because:
- I think sticking to this "generic name" is more consistent with naming style in DVC
- was @dberenbaum 's suggestion in the issues [comment](https://github.com/iterative/dvc/issues/5387#issuecomment-1048026717)
- intuitive short form `-f` (`-o` taken)

Reply:
No strong opinion, but I would also name the internal arguments `force`, I think it is not only the DVC naming style but very common in most CLI tools (https://clig.dev/#arguments-and-flags)

Reply:
In any case, it should be `overwrite` instead of `override`, right?

Reply:
👍  I've added support to `dvc add [target] -o [file] --force` with consistent behavior.